.github/workflows/close-completed-ocwm.yml

```diff
@@ -11,10 +11,10 @@ jobs:
   close-issues:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
 
     - name: Set up Node 18
-      uses: actions/setup-node@v2
+      uses: actions/setup-node@v4
       with:
         node-version: '18'
```

Comment:
Use node-version 20

Reply:
Changed the node version at appropriate places.